assets/src/data/content/resource.ts

```diff
@@ -215,6 +249,21 @@ export interface GroupContent {
   children: Immutable.List<ResourceContent>;
 }
 
+export interface AlternativesContent {
+  type: 'alternatives';
+  id: string;
+  strategy: AlternativesStrategy;
+  children: Immutable.List<AlternativeContent>;
+  alternatives_id: number;
```

Comment:
Unfortunately the clash of schema/server-side conventions means that this variable `alternatives_id` must be snake-cased instead of camel